WordPress/src/main/java/org/wordpress/android/ui/reader/repository/usecases/GetDiscoverCardsUseCase.kt

```diff
@@ -39,8 +44,13 @@ class GetDiscoverCardsUseCase @Inject constructor(
                             ReaderConstants.JSON_CARD_POST -> {
                                 // TODO we might want to load the data in batch
                                 val (blogId, postId) = parseDiscoverCardsJsonUseCase.parseSimplifiedPostCard(cardJson)
-                                val post = ReaderPostTable.getBlogPost(blogId, postId, false)
-                                cards.add(ReaderPostCard(post))
+                                val post = readerPostTableWrapper.getBlogPost(blogId, postId, false)
+                                if (post != null) {
+                                    cards.add(ReaderPostCard(post))
+                                } else {
+                                    appLogWrapper.d(READER, "Post from /cards json not found in local db")
```

Comment:
Since we have multiple databases in the app how about saying: 
```suggestion
                                    appLogWrapper.d(READER, "Post from /cards json not found in ReaderDatabase")
```

Reply:
Fixed in 66f7a9314c5d6d620387ca60ff34466f69fb0f42